log/go.mod

```diff
@@ -3,6 +3,7 @@ module cosmossdk.io/log
 go 1.20
 
 require (
+	github.com/pkg/errors v0.9.1
```

Comment:
This is a archived dependency. Any chance we can use std lib or hashicorp errors

Reply:
It was based on nhttps://github.com/rs/zerolog#error-logging-with-stacktrace.
I'll create a custom `ErrorStackMarshaler` then